opentech/public/funds/templates/public_funds/lab_page.html

```diff
@@ -3,14 +3,27 @@
 
 {% block content %}
     {% include "public_funds/includes/lab_apply_cta.html" %}
-    <div class="wrapper wrapper--flex">
+    <div class="wrapper">
         <section class="section section--main">
-            <h1>{{ page.icon }}{{ page.title }}</h1>
-            <h5>{{ page.introduction }}</h5>
+            <div class="media-box media-box--reverse">
```

Comment:
When no icon or intro are set, we'll get a funky empty set of divs. I say less markup, the better

Reply:
Yeah, Chris should hopefully clear that up. It came from a stale branch so probably isnt correct.